### DISCUSSION TITLE
2FA QR code is not scannable

### DISCUSSION THREAD

**Discussion by thany** [community/community#30389] [Other Feature Feedback, Questions, & Ideas]
The QR-code for activating an authenticator app is not scannable. All other sites provide a scannable code, but Github does not.

It might require a white/light background, but the end result is that I had to enter a code manually which is not a great experience in comparison. Can somebody please get this looked at?

**Comment by joshuawalker** [comment]
Thanks for letting us know! We've shipped a fix for this.
